app/controllers/rx_controller.rb

```diff
@@ -4,13 +4,21 @@
 class RxController < ApplicationController
   include ActionController::Serialization
 
-  skip_before_action :authenticate
+  before_action :authorize_rx
   before_action :authenticate_client
 
   protected
 
   def client
-    @client ||= Rx::Client.new(session: { user_id: ENV['MHV_USER_ID'] })
+    @client ||= Rx::Client.new(session: { user_id: mhv_correlation_id })
+  end
+
+  def authorize_rx
+    mhv_correlation_id || raise(Common::Exceptions::Unauthorized)
```

Comment:
I would agree in using 404 for specific resources eg `prescriptions/1` to avoid leaking information about existence of a resource, but this is denying access to anything within `prescriptions` without even checking for the presence of a resource. It isn't entirely clear, but it seems like 401 is for when a user is not authenticated or authentication failed, and 403 is for when a user is authenticated but not authorized. It looks like there are different interpretations of when to use each, with some favoring 401 even if authentication passes. I am OK with that, we should just make sure the decision is documented somewhere so it can be part of a larger API doc at some point